scripts/deploy/generate-yamls.sh

```diff
@@ -80,6 +81,10 @@ while [ "$#" -gt 0 ]; do
       shift
       tag=$1
       ;;
+    -n)
```

Comment:
we should add this to the help usage above

Reply:
Done. Thanks.